interview-questions/activity-interview-questions.md

```diff
@@ -0,0 +1,70 @@
+# Activity: Interview Questions
+
+## Goal
+
+Our goal is to compile a series of interview question resources and interview questions. By the end of this activity, everyone should have access to some recommended interview questions.
+
+## Preparation
+
+The facilitator should make a spreadsheet that includes everyone's name, and allows students to notate at least two questions. For example:
+
+| Name | Question #1 | Question #2 |
+| ---- | ----------- | ----------- |
+|      |             |
+
+The facilitator should also prepare a way to record ideas in part one. Some recommended methods include:
+
+- The word cloud feature of [Mentimeter](https://www.mentimeter.com/)
+- Taking notes on a shared document
+
+## Activity Instructions
+
+There are two parts to this activity:
+
+1. Crowdsource interview resources
+1. Crowdsource interview questions
+
+### Crowdsource Resources
+
+As a large group, follow directions from the facilitator to create a shared list of interview question resources.
+
+These resources may include:
+
+- Websites made for technical interview preparation
+- Newsletters made for technical interview preparation
+- Recommended books
+
+If it includes practice interview questions, then it's valid!
+
+### Crowdsource Questions
+
+Afterwards, individually find and share practice questions. Everyone must contribute at least _two_ questions to the shared document of practice questions.
```

Comment:
Afterwards is unnecessary. The document is presented in order, and this is already in a new section. As a linking phrase between paragraphs in the same section, it would be more helpful (N.B. American English prefers afterward to afterwards), but here, It feels OK to cut to me.
```suggestion
Individually find and share practice questions. Everyone must contribute at least _two_ questions to the shared document of practice questions.
```

Reply:
TIL "afterward" vs "afterwards"

Thank you!